src/__tests__/helpers/test-utils.tsx

```
@@ -0,0 +1,164 @@
+import { render, RenderOptions } from '@testing-library/react'
+import { NextIntlClientProvider } from 'next-intl'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { AuthProvider } from '@/contexts/auth-context'
+import { ThemeProvider } from '@/lib/theme-provider'
+import type { User, Session } from '@supabase/supabase-js'
+
+interface CustomRenderOptions extends Omit<RenderOptions, 'wrapper'> {
+  initialSession?: Session | null
+  user?: User | null
+  locale?: string
+  messages?: Record<string, any>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)